src/frontend/src/tests/lib/components/agreements/AcceptAgreementsCheckbox.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { cleanup, fireEvent, render } from '@testing-library/svelte';
+import Host from './AcceptAgreementsCheckboxHost.svelte';
+
+// --- Mock i18n store so $i18n.* is available (typed, no any) -----------
+vi.mock('$lib/stores/i18n.store', async () => {
+	const { readable } = await import('svelte/store');
+	const i18n = readable({
+		agreements: {
+			text: {
+				i_have_accepted: 'I have accepted',
+				i_have_accepted_updated: 'I have accepted (updated)'
+			}
+		}
+	});
+	return { i18n };
+});
+
```

Comment:
you can use the mock


```suggestion
import Host from './AcceptAgreementsCheckboxHost.svelte';
import en from '$tests/mocks/i18n.mock';

```